{"uid":"25aed99b3c1e1fa5","name":"Administrador acessa a aplicação visualização das telas","fullName":"Gerenciamento de Perfis de Acesso: Administrador acessa a aplicação visualização das telas","historyId":"d721f7d5249fd2b39be5431b0a67cc75","time":{"start":1744137947397,"stop":1744137947445,"duration":48},"status":"broken","statusMessage":"AttributeError: 'Context' object has no attribute 'perfil_page'\n","statusTrace":"  File \"C:\\Users\\mferreio\\OneDrive - NTT DATA EMEAL\\Documents\\__Github_Projetos__\\AutomacaoDeTeste\\venv\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\mferreio\\OneDrive - NTT DATA EMEAL\\Documents\\__Github_Projetos__\\AutomacaoDeTeste\\venv\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\test_perfil_de_acesso_steps.py\", line 8, in step_then_acesso_total\n    assert context.perfil_page.possui_acesso_total(), \"Usuário não possui acesso total ao sistema.\"\n           ^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\mferreio\\OneDrive - NTT DATA EMEAL\\Documents\\__Github_Projetos__\\AutomacaoDeTeste\\venv\\Lib\\site-packages\\behave\\runner.py\", line 321, in __getattr__\n    raise AttributeError(msg)\n","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"broken","statusMessage":"AttributeError: 'Context' object has no attribute 'perfil_page'\n","statusTrace":"  File \"C:\\Users\\mferreio\\OneDrive - NTT DATA EMEAL\\Documents\\__Github_Projetos__\\AutomacaoDeTeste\\venv\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\mferreio\\OneDrive - NTT DATA EMEAL\\Documents\\__Github_Projetos__\\AutomacaoDeTeste\\venv\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\test_perfil_de_acesso_steps.py\", line 8, in step_then_acesso_total\n    assert context.perfil_page.possui_acesso_total(), \"Usuário não possui acesso total ao sistema.\"\n           ^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\mferreio\\OneDrive - NTT DATA EMEAL\\Documents\\__Github_Projetos__\\AutomacaoDeTeste\\venv\\Lib\\site-packages\\behave\\runner.py\", line 321, in __getattr__\n    raise AttributeError(msg)\n","steps":[{"name":"Given que o usuário está logado como \"Administrador\"","time":{"start":1744137947401,"stop":1744137947433,"duration":32},"status":"passed","steps":[{"name":"Validando que o usuário está logado como Administrador","time":{"start":1744137947402,"stop":1744137947433,"duration":31},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"context","value":"<behave.runner.Context object at 0x00000262475381A0>"}],"stepsCount":0,"attachmentStep":false,"hasContent":true,"shouldDisplayMessage":false,"attachmentsCount":0}],"attachments":[],"parameters":[],"stepsCount":1,"attachmentStep":false,"hasContent":true,"shouldDisplayMessage":false,"attachmentsCount":0},{"name":"Then eu devo ter acesso total ao sistema","time":{"start":1744137947433,"stop":1744137947441,"duration":8},"status":"broken","statusMessage":"AttributeError: 'Context' object has no attribute 'perfil_page'\n","statusTrace":"  File \"C:\\Users\\mferreio\\OneDrive - NTT DATA EMEAL\\Documents\\__Github_Projetos__\\AutomacaoDeTeste\\venv\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\mferreio\\OneDrive - NTT DATA EMEAL\\Documents\\__Github_Projetos__\\AutomacaoDeTeste\\venv\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\test_perfil_de_acesso_steps.py\", line 8, in step_then_acesso_total\n    assert context.perfil_page.possui_acesso_total(), \"Usuário não possui acesso total ao sistema.\"\n           ^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\mferreio\\OneDrive - NTT DATA EMEAL\\Documents\\__Github_Projetos__\\AutomacaoDeTeste\\venv\\Lib\\site-packages\\behave\\runner.py\", line 321, in __getattr__\n    raise AttributeError(msg)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentStep":false,"hasContent":true,"shouldDisplayMessage":true,"attachmentsCount":0},{"name":"And eu devo conseguir visualizar todas as telas","time":{"start":1744137947445,"stop":1744137947445,"duration":0},"status":"skipped","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentStep":false,"hasContent":false,"shouldDisplayMessage":false,"attachmentsCount":0}],"attachments":[],"parameters":[],"stepsCount":4,"attachmentStep":false,"hasContent":true,"shouldDisplayMessage":false,"attachmentsCount":0},"afterStages":[],"labels":[{"name":"severity","value":"normal"},{"name":"feature","value":"Gerenciamento de Perfis de Acesso"},{"name":"framework","value":"behave"},{"name":"language","value":"cpython3"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":true,"retry":true,"extra":{"categories":[],"tags":[]},"source":"25aed99b3c1e1fa5.json","parameterValues":[]}